zproject/local_settings_template.py

```diff
@@ -148,39 +151,47 @@
 # The email gateway provides, for each stream, an email address that
 # you can send email to in order to have the email's content be posted
 # to that stream.  Emails received at the per-stream email address
-# will be converted into a Zulip message
-
+# will be converted into a Zulip message.
+#
 # There are two ways to make use of local email mirroring:
 #  1. Local delivery: A MTA runs locally and passes mail directly to Zulip
 #  2. Polling: Checks an IMAP inbox every minute for new messages.
-
-# A Puppet manifest for local delivery via Postfix is available in
-# puppet/zulip/manifests/postfix_localmail.pp. To use the manifest, add it to
-# puppet_classes in /etc/zulip/zulip.conf. This manifest assumes you'll receive
-# mail addressed to the hostname of your Zulip server.
-#
-# Users of other mail servers will need to configure it to pass mail to the
-# email mirror; see `python manage.py email-mirror --help` for details.
-
-# The email address pattern to use for auto-generated stream emails
-# The %s will be replaced with a unique token, and the resulting email
-# must be delivered to the EMAIL_GATEWAY_IMAP_FOLDER of the
-# EMAIL_GATEWAY_LOGIN account below, or piped in to the email-mirror management
-# command as indicated above.
 #
-# Example: zulip+%s@example.com
-EMAIL_GATEWAY_PATTERN = ""
-
-
-# The following options are relevant if you're using mail polling.
+# The local delivery configuration is preferred for production because
+# it doesn't have delay, while the polling mechanism is preferred for
```

Comment:
This is again, probably overkill, but for me (a noob) reading this, I read "is preferred for development" as meaning that it's preferred by the project maintainers in some way when I think what is meant is "most people doing development prefer setting up an IMAP mailbox and polling it" (the former sounding like a prescription, the latter sounding like advice/guidance)